### PR TITLE
Added Roadmap and Future plans to README and Fixed issues #12

### LINKED ISSUE #5: Create Road Map and Future plans in README

### PR DESCRIPTION
**Pull request by harsh-jindal**
Fix #5 
Changes: Added Roadmap and Future plans to README.md

### CODE REVIEW

**Comment by srv-twry** [comment]
@harsh-jindal It would be better if you could add bullet points instead of writing all the features separated by commas.
As a side note, always try to solve one issue per PR.

**Comment by harsh-jindal** [comment]
I'll do it.

**Comment by harsh-jindal** [comment]
How to make separate pull requests?

**Comment by ShikharJ** [comment]
@harsh-jindal On your current branch, create another branch and give it a name. Then `drop` the commit you don't need and send a PR through that branch. Refer to `git` rebasing for clarification.

**Comment by harsh-jindal** [comment]
Okay

File: .gitignore [hunk 1]
```diff
@@ -1,9 +1,7 @@
 *.iml
 .gradle
 /local.properties
-/.idea/workspace.xml
-/.idea/libraries
-/.idea/misc.xml
+.idea
```
Comment (sairamch04) [inline .gitignore]: This does not work unless `.idea` is removed from index. 
Use either `git rm --cached .idea`(preferred) or just `rm -rf .idea` and commit

Comment (harsh-jindal) [reply .gitignore]: Okay I'll do it.

**Pull request closed without merging**